frontend/jest.config.js

```diff
@@ -23,5 +23,5 @@ module.exports = {
   testMatch: ['**/?(*.)+(spec|test).js?(x)'],
   // snapshotSerializers: ['jest-serializer-vue'],
   transformIgnorePatterns: ['<rootDir>/node_modules/(?!vee-validate/dist/rules)'],
-  testEnvironment: 'jest-environment-jsdom-sixteen',
+  // testEnvironment: 'jest-environment-jsdom-sixteen', // not needed anymore since jest@26, see: https://www.npmjs.com/package/jest-environment-jsdom-sixteen
```

Comment:
I allowed my self to remove this.
Especally in the hope it makes my date tests working with `jest
  .useFakeTimers()
  .setSystemTime()`. But it unfortunatelly did not help.

Reply:
Did followup issue #2128